packages/components/src/util/typography.module.css

```
@@ -0,0 +1,94 @@
+/* sizes */
+
+.typography,
+.typography--size-body {
+  @apply text-body leading-body;
```

I asked @dcwither this before, but don't remember the answer 🤦 - can we not use the global Tailwind classes and get the performance / bundle size benefits of Tailwind?

To use those, this would look something like

```suggestion
  composes: text-body leading-body from global;
```

---

The difficulty is that in an unknown number of cases bootstrap has its own utility classes which can conflict with these. It's definitely something to look into, but right now I'm worried about those overriding these styles in unpredictable ways.